python/lsst/pipe/supertask/super_task.py

```diff
@@ -161,6 +166,54 @@ def makeArgumentParser(cls):
         parser.add_id_argument("--id", "raw", help="data IDs, e.g. --id visit=12345 ccd=1,2^0,3")
         return parser
 
+    @classmethod
+    def makeTargetList(cls, refList, kwargs={}):
```

Comment:
Don't use a mutable datatype as an input argument; the default value kwargs can be modified in a way that will persist the next time the function is called. 
Instead say `kwargs=None` and then later say `if kwargs is None: kwargs = {}`
And as above, kwargs seems like a misleading name, IMHO.